.github/actions/ipsec-key-rotate/action.yaml

```diff
@@ -70,13 +62,9 @@ runs:
           sleep 30s
         done
 
-        exp_nb_keys=1
-        if [[ "${{ inputs.key-type-two }}" == "+" ]]; then
-          exp_nb_keys=8
-          if [[ "${{ inputs.encryption-overlay }}" == "true" ]]; then
-            exp_nb_keys=12
-          fi
-        fi
+        # After the key rotation is finished and old keys are cleaned up, the number of keys is
+        # halved.
+        ((exp_nb_keys/=2))
```

Comment:
How come this wasn't failing before?

Reply:
If you look at the numbers, they were also halved before. It just was hardcoded.